README.rst

```diff
@@ -671,7 +746,31 @@ Preprocessed images
 Group-level
 ===========
 
--  ``grouplevel/...``
+-  |  For task-based group-level analyses, HALFpipe outputs the statistical maps for the effect, the variance, the degrees of freedom of the variance, and the t-statistic and z-statistic for each linear model that is specified, much as for the first-level features. These models are fit per session, task, first-level feature, first-level task contrast, and group-level contrast, where these exist.
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-effect_statmap.nii.gz``
```

Comment:
We only have `taskcontrast` for task-based data, so I would leave that out here

Reply:
Does that also mean that the contrast- part of the filename does not exist for data other than task-based?